packages/email/template-components/template-document-invite.tsx

```diff
@@ -19,17 +20,30 @@ export const TemplateDocumentInvite = ({
   signDocumentLink,
   assetBaseUrl,
   role,
+  selfSigner,
 }: TemplateDocumentInviteProps) => {
   const { actionVerb, progressiveVerb } = RECIPIENT_ROLES_DESCRIPTION[role];
+  const headerContent = selfSigner ? (
+    <>
+      {`Please ${actionVerb.toLowerCase()} your document`}
+      <br />
+      {`"${documentName}"`}
+    </>
+  ) : (
+    <>
+      {`${inviterName} has invited you to ${actionVerb.toLowerCase()}`}
+      <br />
+      {`"${documentName}"`}
+    </>
+  );
```

Comment:
Can you put this directly in the template instead?

Reply:
done